keyboards/choc_taro/matrix.c

```diff
@@ -0,0 +1,287 @@
+/*
```

Comment:
Why the custom matrix?

Reply:
This keyboard uses col2rol2col.
I used custom matrix for this.

Reply:
Ah, okay then. In that case, you can eliminate a lot of the duplicate code here by using Custom Matrix Lite: https://docs.qmk.fm/#/custom_matrix?id=lite

Reply:
okay.